src/components/LinkWithLang.js

```diff
@@ -35,6 +35,8 @@ class LinkWithLangComponent extends React.Component {
 
 LinkWithLangComponent.propTypes = {
   to: PropTypes.shape({
+    hash: PropTypes.string.isRequired,
```

Comment:
Got this error `Warning: Failed prop type: The prop to.hash is marked as required in LinkWithLangComponent`. Looks like it's missing somewhere in Navbar